lang/es.json

```diff
@@ -0,0 +1,955 @@
+        "password": "Debe introducir una contraseña.",
+        "password_match": "Sus contraseñas no coinciden.",
+        "invalid_password": "Las contraseñas deben tener al menos 7 caracteres e incluir caracteres alfabéticos y numéricos.",
+        "field_not_blank": " el campo no puede dejarse en blanco.",
+        "certificate_amount": "Debe introducir un importe para el cupón de regalo.",
+        "certificate_amount_range": "Debe introducir un importe de certificado entre [MIN] y [MAX]",
+        "price_min_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_max_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_min_not_entered": "El campo de precio mínimo es obligatorio.",
+        "price_max_not_entered": "El campo de precio máximo es obligatorio.",
+        "price_invalid_value": "La entrada debe ser mayor que 0."
+    },
+    "validation_fallback_messages": {
+        "valid_email": "Debe introducir un correo electrónico válido.",
+        "password": "Debe introducir una contraseña.",
+        "password_match": "Sus contraseñas no coinciden.",
+        "invalid_password": "Las contraseñas deben tener al menos 7 caracteres e incluir caracteres alfabéticos y numéricos.",
+        "field_not_blank": " el campo no puede dejarse en blanco.",
+        "certificate_amount": "Debe introducir un importe para el cupón de regalo.",
+        "certificate_amount_range": "Debe introducir un importe de certificado entre [MIN] y [MAX]",
+        "price_min_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_max_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_min_not_entered": "El campo de precio mínimo es obligatorio.",
+        "price_max_not_entered": "El campo de precio máximo es obligatorio.",
+        "price_invalid_value": "La entrada debe ser mayor que 0."
+    },
+    "validation_default_messages": {
+        "valid_email": "Debe introducir un correo electrónico válido.",
+        "password": "Debe introducir una contraseña.",
+        "password_match": "Sus contraseñas no coinciden.",
+        "invalid_password": "Las contraseñas deben tener al menos 7 caracteres e incluir caracteres alfabéticos y numéricos.",
+        "field_not_blank": "El campo no puede dejarse en blanco.",
+        "certificate_amount": "Debe introducir un importe para el cupón de regalo.",
+        "certificate_amount_range": "Debe introducir un importe de certificado entre [MIN] y [MAX]",
+        "price_min_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_max_evaluation": "El precio mínimo debe ser menor que el precio máximo.",
+        "price_min_not_entered": "El campo de precio mínimo es obligatorio.",
+        "price_max_not_entered": "El campo de precio máximo es obligatorio.",
+        "price_invalid_value": "La entrada debe ser mayor que 0."
+    }
+}
```

Comment:
should we add a new line at the end of file?

Reply:
updated.